doc/library/index.txt

```diff
@@ -27,6 +27,7 @@ Types and Ops that you can use to build and compile expression graphs.
    sparse/sandbox
    tensor/index
    typed_list
+   tests
```

Comment:
It gives the following:
```
Warning, treated as error:
/home/jenkins-cpu1/workspace/Theano PR multijob/doc/library/index.txt:12: WARNING: toctree contains reference to nonexisting document u'library/tests'
```
I don't see any file named `tests.txt` in `doc/library` in the repo. Do you have it locally? Should you commit it?